index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <h1>Hello Microverse</h1>
+  </body>
+</html>
```

Comment:
- [ ] You didn't add an MIT.md file which is basically your license file. Please add it using this [template](https://github.com/microverseinc/readme-template/blob/master/MIT.md) :blush:
- [ ] Don't forget to add your name, current year, app type in your MIT.md file after adding it in your repository because most of the students forget this part :smile: